README.md

```diff
@@ -1,9 +1,16 @@
 # PageObjectExtension
 
-Provides default classes for Page object pattern in Behat:
+Provides default classes for Page object pattern in Behat.
 
-* `Page` - represents specific page on web, API etc.
-* `Element` - represents part of the page
+### Page
+
+`Page` represents specific page on web, API etc.
+This concept is extracted from [Sylius Behat system](https://github.com/Sylius/Sylius/tree/master/src/Sylius/Behat/Page) and
+inspired by [sensiolabs/BehatPageObjectExtension](https://github.com/sensiolabs/BehatPageObjectExtension/tree/master/src/PageObject)
+
+### Element
+
+`Element` represents part of the page. This concept is extracted from [URBANARA CatalogPromotion plugin](https://github.com/URBANARA/UrbanaraCatalogPromotionPlugin/tree/master/tests/Behat/Element).
```

Comment:
I would skip `This concept is extracted from [URBANARA CatalogPromotion plugin](https://github.com/URBANARA/UrbanaraCatalogPromotionPlugin/tree/master/tests/Behat/Element).` part